packages/atomic/src/components/search/atomic-facet-manager/atomic-facet-manager.tsx

```diff
@@ -20,7 +20,8 @@ import {
 import {Bindings} from '../atomic-search-interface/atomic-search-interface';
 
 /**
- * The `atomic-facet-manager` helps reorder facets and their values to match the most recent search response with the most relevant results. A facet component is slotted within an `atomic-facet-manager` to leverage this functionality.
+ * The `atomic-facet-manager` helps reorder facets and their values to match the most recent search response with the most relevant results.
```

Comment:
@jpmarceau 👁️

Reply:
Once generated, here's what the effect of this annotation would be:
![Screenshot 2024-05-27 at 4 09 38 PM](https://github.com/coveo/ui-kit/assets/39384459/4954cbf1-7ee0-40e8-bc06-d550ffeef0f1)
I haven't looked into the slot functionality in depth, but this looks good to me?

Reply:
Yep it make sense 👍